htdocs/modules/system/themes/modern/widgets/realestate/class/ModernThemeWidget.php

```
@@ -0,0 +1,99 @@
+        return [
+            'title'     => 'Real Estate',
+            'icon'      => '🏠',
+            'stats'     => [
+                'for_sale' => (int) $forSale,
+                'for_rent' => (int) $forRent,
+                'closed'   => (int) $closed,
+            ],
+            'recent'    => $recent,
+            'admin_url' => XOOPS_URL . '/modules/realestate/admin/',
+        ];
+    }
```

PHPMD (2.15.0)

[warning] The method getWidgetData() has a Cyclomatic Complexity of 10. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)